HandsOn/Group11/mappings/RML_operations-with-links.rml

```diff
@@ -0,0 +1,169 @@
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtist ;
+        rr:object [
+            rr:reference "artist" ;
+            rr:datatype xsd:string;
+            ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtistRole ;
+        rr:object [
+            rr:reference "artistRole";
+            rr:datatype xsd:string;
+             ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtistId ;
+        rr:object [
+            rr:reference "artistId";
+            rr:datatype xsd:interger; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasTitle ;
+        rr:object [
+            rr:reference "title";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasMedium ;
+        rr:object [
+            rr:reference "medium";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasCreditLine ;
+        rr:object [
+            rr:reference "creditLine";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art::madeInYear ;
+        rr:object [
+            rr:reference "year";
+            rr:datatype xsd:integer; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art::acquiredInYear ;
+        rr:object [
+            rr:reference "acquisitionYear";
+            rr:datatype xsd:integer; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasThumbnailCopyright ;
+        rr:object [
+            rr:reference "thumbnailCopyright";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasThumbnailUrl ;
+        rr:object [
+            rr:reference "thumbnailUrl";
+            rr:datatype xsd:anyURI;
+         ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasUrl ;
+        rr:object [
+            rr:reference "url";
+            rr:datatype xsd:anyURI; ];
+    ].
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasDimension ;
+        rr:object [
+            rr:reference "http://artwork.org/artwork/{id}/dimensions"; ];
+    ].
+
+    rr:predicateObjectMap [
+        rr:predicate art:sameAsDbpediaArtist ;
+        rr:object [
+            rr:reference "same_as_dbpedia_artist";
+            rr:datatype xsd:anyURI; ];
+    ].
+
+
+<#DimensionTriplesMap>
+    rml:logicalSource [
+    rml:source <https://drive.google.com/file/d/12ZgCUhrQx0pGQN3jjTdLPUmvOPbJP0gQ/view?usp=share_link>;
+    rml:referenceFormulation ql:CSV;
+    rml:iterator "csvRow";
+];
+
+
+    rr:subjectMap [
+        rr:template "http://artwork.org/artwork/{id}/dimensions" ;
+        rr:termType rr:IRI;
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasWidth ;
+        rr:object [
+            rr:reference "width";
+            rr:datatype xsd:integer; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasHeight ;

```

Comment:
hasDimensionsUnit? hasHeight is defined below

Reply:
I think we can skip it anyway, we assume it's always gonna be "mm" so there's no need to keep it in the object in my opinion